README.md

```diff
@@ -76,10 +77,17 @@ mvn install
 ````
 
 ### Compile & Build the Proxy
+
 ```bash
 git clone https://github.com/clarus-proxy/proxy.git
 cd proxy
 git checkout develop
-cd proxy
```

Comment:
Just corrected in the commit 405523c975600735b81aef0de90eacc94309a989

Reply:
Nice thanks